src/main/java/de/orchound/gameoflife/GameOfLifeApplet.java

```diff
@@ -99,17 +101,27 @@ private void drawHud() {
 	public void mouseDragged() {
 		if (mouseButton == LEFT) {
 			viewOffset.add(mouseX - pmouseX, mouseY - pmouseY);
+		} else if (mouseButton == RIGHT) {
+			try {
+				Vector2f position = bufferVector2f.set(mouseX, mouseY)
+					.sub(viewOffset)
+					.div(scale);
+				Vector2i cell = boardView.getCellAt(position, bufferVector2i);
+				game.setCell(cell);
+			}  catch (ArrayIndexOutOfBoundsException ignored) {}
```

Comment:
Don't catch and ignore these errors, prevent them from happening.
Just as done in `Game.toggleCell()`, do input validation in `Game.setCell()`.